plugin-cgroupv2/README.md

```diff
@@ -1,8 +1,14 @@
-# Kubernetes plugin
+# Cgroup V2 plugins
 
-Allows to measure K8S pods using the cgroup v2 sysfs.
+Allows to measure CPU-time used using the cgroup v2 sysfs.
 
-# How to use
+# Table of Contents
```

Comment:
Simple question, why are you using \# which is meant to be used for the header ?

Reply:
idk :/

Reply:
I don't think that this has been resolved

Reply:
Changed from "#" to "##"